ui/src/common/routes.ts

```diff
@@ -23,10 +23,9 @@ export const USER_SIGNUP = `${USER}/signup`;
 export const USER_LOCAL_LOGIN = `${USER_LOGIN}/local`;
 export const USER_SETTINGS = `${USER}/settings`;
 
-// TODO: rename `holdingpen` to `admin`
 export const HOLDINGPEN = '/holdingpen';
 export const HOLDINGPEN_DASHBOARD = `${HOLDINGPEN}/dashboard`;
-export const HOLDINGPEN_INSPECT = `${HOLDINGPEN}/inspect`;
```

Comment:
actually, we have to keep the inspect, is a tool for curators. Any particular reason why we removed it?

Reply:
I thought it's not needed, I'll revert :)